loop/mi_views.py

```diff
@@ -0,0 +1,189 @@
+import os
+import json
+
+import requests
+from django.http import HttpResponse
+
+from loop_ivr.outliers.removal import remove_crop_outliers
+from loop_ivr.helper_function import run_query
+from loop_ivr.utils.marketinfo import get_query
+
+from mi_data_structure import *
+from loop.models import *
+import pandas as pd
+from tastypie.models import ApiKey
+from crop_price_structure import *
+from loop.utils.mi_pilot_var import mandi_list, agg_list
+
+fileDir = os.path.dirname(os.path.realpath('__file__'))
+
+def get_aggregator_mi_related_data(request):
+    
+    agg_list_requested = is_authenticated(request)
+    
+    if agg_list_requested :
+        
+        # Create Objects
+        agg_data_obj = LoopUser.objects.filter(user__in=agg_list_requested).values('user', 'name_en', 'preferred_language_id')
+
+        # get Dict from Single Obj
+        agg_obj = agg_data_obj[0]
+        preferred_lang_suffix = '_en' if agg_obj['preferred_language_id'] == 2 else ''
+
+        # Mandi Name in preferred lang of aggregator
+        mandi_name_label = 'mandi_name' + preferred_lang_suffix
+        mandi_data_obj = Mandi.objects.filter(id__in=mandi_list).values('id', mandi_name_label)
+
+        #Gaddidar Name in preferred lang of aggregator
+        gaddidar_name_label = 'gaddidar_name' + preferred_lang_suffix
+        gaddidar_data_obj = Gaddidar.objects.filter(mandi__id__in=mandi_list)
+
+        # Read CSV file and filter for requested Aggreagator
+        filepath = os.path.join(fileDir, "/home/trionfo/Documents/dg/loop/utils/Transport Cost & Capacity Data - Sheet1.csv")
+        print filepath
+        transport_dataframe = pd.read_csv(filepath)
+        transport_dataframe = transport_dataframe[transport_dataframe['Aggregator Id']==agg_list_requested[0]]
+
+        # Store Json Result
+        agg_res = []
+
+        # Mandi Detail
+        mandi_res = []    
+        for mandiobj in mandi_data_obj:
+            mandiobjdetail = MandiDetail(mandi_id=mandiobj['id'], mandi_name=mandiobj[mandi_name_label], mandi_category='Chhoti Mandi',\
+                        mandi_distance='590')
```

Comment:
why are category and distance hardcoded?

Reply:
For Initially, testing.